automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -44,53 +48,120 @@ object GPAllocFixtureSpec {
    */
-  private def claimProject(): GoogleProject = {
-    val claimedBillingProject = claimGPAllocProject(hermioneCreds)
-    Orchestration.billing.addUserToBillingProject(claimedBillingProject.projectName,
-                                                  ronEmail,
-                                                  BillingProject.BillingProjectRole.User)(hermioneAuthToken)
-    logger.info(s"Billing project claimed: ${claimedBillingProject.projectName}")
-    GoogleProject(claimedBillingProject.projectName)
-  }
+  private def claimProject(): IO[GoogleProject] =
+    for {
+      claimedBillingProject <- IO(claimGPAllocProject(hermioneCreds))
+      _ <- IO(
+        Orchestration.billing.addUserToBillingProject(claimedBillingProject.projectName,
+                                                      ronEmail,
+                                                      BillingProject.BillingProjectRole.User)(hermioneAuthToken)
+      )
+      _ <- IO(logger.info(s"Billing project claimed: ${claimedBillingProject.projectName}"))
+    } yield GoogleProject(claimedBillingProject.projectName)
 
   /**
    * Unclaiming billing project claim by Hermione
    */
-  private def unclaimProject(project: GoogleProject): Unit = {
-    Orchestration.billing.removeUserFromBillingProject(project.value, ronEmail, BillingProject.BillingProjectRole.User)(
-      hermioneAuthToken
-    )
-    releaseGPAllocProject(project.value, hermioneCreds)
-    logger.info(s"Billing project released: ${project.value}")
-  }
+  private def unclaimProject(project: GoogleProject): IO[Unit] =
+    for {
+      _ <- IO(
+        Orchestration.billing
+          .removeUserFromBillingProject(project.value, ronEmail, BillingProject.BillingProjectRole.User)(
+            hermioneAuthToken
+          )
+      )
+      _ <- IO(releaseGPAllocProject(project.value, hermioneCreds))
+      _ <- IO(logger.info(s"Billing project released: ${project.value}"))
+    } yield ()
+
+  private def createInitialRuntime(project: GoogleProject): IO[Unit] =
+    LeonardoApiClient.client.use { c =>
+      implicit val authHeader = Authorization(Token(AuthScheme.Bearer, ronCreds.makeAuthToken().value))
+      implicit val client = c
+      for {
+        res <- LeonardoApiClient
+          .createRuntimeWithWait(
+            project,
+            initalRuntimeName,
+            LeonardoApiClient.defaultCreateRuntime2Request
+          )
+          .attempt
+        _ <- res match {
+          case Right(_) =>
+            IO(logger.info(s"Created initial runtime ${project.value} / ${initalRuntimeName.asString}"))
+          case Left(err) =>
+            IO(logger.warn(
+                 s"Failed to create initial runtime ${project.value} / ${initalRuntimeName.asString} with error"
+               ),
+               err)
+        }
+      } yield ()
+    }
+
+  private def deleteInitialRuntime(project: GoogleProject): IO[Unit] =
+    LeonardoApiClient.client.use { c =>
```

Comment:
fyi, you can do `implicit c =>` directly so that you don't have to assign it to an implicit variable later